Horion/DrawUtils.h

```diff
@@ -129,6 +129,8 @@ class DrawUtils {
 	static void drawNameTags(C_Entity* ent, float textSize, bool drawHealth = false, bool useUnicodeFont = false);
 	static void drawItem(C_ItemStack* item, const vec2_t& ItemPos, float opacity, float scale, bool isEnchanted);
 	static void drawKeystroke(char key, const vec2_t& pos);
+	static void drawLeftMouseKeystroke(vec2_t pos);
+	static void drawRightMouseKeystroke(vec2_t pos);
```

Comment:
no

Reply:
move these to the HudModule class please, drawKeystroke should never have been inside DrawUtils in the first place